pkg/3scale/amp/component/apicast.go

```diff
@@ -16,6 +18,9 @@ import (
 const (
 	ApicastStagingName    = "apicast-staging"
 	ApicastProductionName = "apicast-production"
+
+	CustomPoliciesMountBasePath    = "/opt/app-root/src/policies"
+	CustomPoliciesAnnotationPrefix = "policy.apicast.3scale.redhat.com/"
```

Comment:
Maybe we should use `apps.3scale.net` as the prefix as in other annotations already set for this operator?
https://github.com/3scale/3scale-operator/blob/master/apis/apps/v1alpha1/apimanager_types.go#L39

Reply:
Updated implementation. Now it is 

PREFIX-HASH: VOLUME_NAME

```
apps.3scale.net/apicast-policy-volume-550ce71bf15f7ab80906b5e066d2fc30: policy-01-exampleother
```

Reply:
:+1: